cmd/notary/prettyprint_test.go

```diff
@@ -167,9 +167,9 @@ func TestPrettyPrintSortedTargets(t *testing.T) {
 		assert.NoError(t, err)
 	}
 	unsorted := []*client.Target{
-		{Name: "zebra", Hashes: data.Hashes{"sha256": hashes[0]}, Length: 8},
-		{Name: "abracadabra", Hashes: data.Hashes{"sha256": hashes[1]}, Length: 1},
-		{Name: "bee", Hashes: data.Hashes{"sha256": hashes[2]}, Length: 5},
+		{Name: "zebra", Role: "targets/b", Hashes: data.Hashes{"sha256": hashes[0]}, Length: 8},
+		{Name: "abracadabra", Role: "targets", Hashes: data.Hashes{"sha256": hashes[1]}, Length: 1},
```

Comment:
It's a unicorn:  http://www.amazon.com/Douglas-Cuddle-Toys-343-Abracadbra/dp/B000C24LPG